modules/nf-core/fastme/tests/main.nf.test

```diff
@@ -110,18 +111,44 @@ nextflow_process {
         when {
             process {
                 """
-                input[0] = TCOFFEE_SEQREFORMAT.out.formatted_file.collect{ meta, aln -> aln }
-                input[1] = []
+                input[0] = TCOFFEE_SEQREFORMAT.out.formatted_file
+                    .map { meta, aln -> [meta, aln, []] }
                 """
             }
         }
 
         then {
             assertAll(
                 { assert process.success },
-                { assert process.out.nwk },
-                { assert process.out.matrix },
-                { assert process.out.bootstrap }
+                { assert process.out.nwk[0][1] =~ ".*.nwk" },
+                { assert process.out.matrix[0][1] =~ ".*.matrix.phy" },
+                { assert process.out.bootstrap[0][1] =~ ".*.bootstrap" },
+                { assert process.out.stats[0][1] =~ ".*_stat.txt" },
+                { assert process.out.versions }
```

Comment:
```suggestion
                { assert path(process.out.nwk[0][1]).text.contains("1atx:") },
                { assert path(process.out.matrix[0][1]).text.contains("1apf") },
                { assert path(process.out.bootstrap[0][1]).text.contains("1atx:") },
                { assert path(process.out.stats[0][1]).readLines()[0..12]).match("stats_boot") },
                { assert snapshot(process.out.versions).match("versions") }
```

I meant something like this. If you look at the [output.zip](https://github.com/nf-core/modules/files/14646877/fastme_out.zip), you can see that some strings are contained across runs and we can check for it 😊. Also don't forget to include a snapshot for the versions. The current test clause, doesn't include any snapshots and it won't be written to the `main.nf.test.snap` file. 

Double check the test for errors, update snapshot and then 🚀